docs/terms/layer.md

```diff
@@ -1,6 +1,13 @@
-page_title: Layers
-page_description: Organizing the Docker Root File System
-page_keywords: containers, lxc, concepts, explanation, image, container
+<!--[metadata]>
++++
+draft = true
```

Comment:
Yes, `draft=true` means they are in the tree but aren't being published.  These aren't.